jax/numpy/lax_numpy.py

```diff
@@ -204,6 +204,13 @@ def _make_scalar_type(np_scalar_type):
 
 _canonicalize_axis = lax._canonicalize_axis
 
+_DEFAULT_TYPEMAP = {
```

Comment:
This change looks unrelated, or at least, I can't see why it would change anything?

Reply:
Yeah - I moved this to a global constant because initially I used it in `arange` as well, before landing on the current approach. I thought it made more sense as a global constant, so I left it.